src/App.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
+import "./App.css";
+import Form from "./components/Form";
+import TaskList from "./components/TaskList";
+import { Task } from "./types";
+
+function App() {
+  const [tasks, setTasks] = useState<Task[]>([]);
+
+  const addTask = (task: Task, callback: () => void) => {
+    const taskWithId = {
+      ...task,
+      id: `${Date.now()}--${Math.random()}`,
+    };
+
+    callback();
+    const updatedTasks = [...tasks, taskWithId];
+
+    setTasks(updatedTasks);
+    localStorage.setItem("tasks", JSON.stringify(updatedTasks));
+  };
+
+  const deleteTask = (TasktoDelete: Task) => {
```

Comment:
I think you can pass just the id here
```suggestion
  const deleteTask = (taskId: string) => {
```

Reply:
Sure, I´ve already modified that.